ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -153,21 +162,20 @@ Other than the _earliest_ (genesis) named block, which MUST be interpreted in th
 > The finality guarantees of Snowman consensus remove any distinction between _safe_ and _finalized_. 
 > Furthermore, the _latest_ block is not at risk of re-org, only of a negligible risk of data corruption local to the API node.
 
-
-### Optimal block building
```

Comment:
I thought we were going to move this to the appendix and change SHOULD to MAY. We should then also add the caveat of it possibly hindering dependent transactions.

Reply:
moved it to the appendix